docs/medical-api-reference/coming-soon.mdx

```diff
@@ -1,7 +1,26 @@
 ---
-title: "Coming Soon!"
-description:
-  "We're hard at work building out our Medical API as you're reading this."
+title: "Medical API"
+description: "Retrieve access to patient's medical records"
 ---
 
-<Warning>Check out our Medical API page for more info and to get in touch: https://metriport.com/medical/</Warning>
+## Request Access
+
+On the Metriport developer dashboard go to the [Medical API Page](https://dash.metriport.com/medical-api), to request access for use of the API.
+Here you can fill out a form that will notify the Metriport team of your request.
+
+You will need to submit:
+
+- Contact information
+- A [BAA](https://healthitsecurity.com/features/what-is-a-hipaa-business-associate-agreement-baa) in PDF format
+- Give confirmation that you are authorized to make requests to the Metriport API on behalf of an entity with an [NPI number](https://www.cms.gov/Regulations-and-Guidance/Administrative-Simplification/NationalProvIdentStand)
+
+<Warning>You will need an active subscription before submitting a request.</Warning>
+
+Once the form is submitted a member from the Metriport team will:
+
+- Review the information
+- Sign the BAA
+- Grant you access to the Medical API
+- Contact you via email with the signed doc stating that your request has been approved
+
+After approval, when you visit the "Medical API" page on the dashboard, you should see (STUFF: What exactly is the first thing they will see?).
```

Comment:
"... you should then be able to access the Medical API admin functionality"

^ the above will be good enough for now - in reality they'll be able to manage Facilities/NPIs/Patients/etc